embedded-memsql/src/test/java/com/playtika/test/memsql/EmbeddedMemSqlBootstrapConfigurationTest.java

```diff
@@ -25,7 +26,9 @@
 @SpringBootTest(classes = EmbeddedMemSqlBootstrapConfigurationTest.TestConfiguration.class,
         properties = {
                 "spring.profiles.active=enabled",
-                "embedded.memsql.install.enabled=true"
+                "embedded.memsql.licenseKey=BGE5YzE5NTdmN2I1NDQ4MjhhNTQ0MTM4YWQ4Y2Q5ZWNiAAAAAAAAAAAEAAAAAAAAAAwwNQIYKbH2LOLeT189H+nBdRagLdLboVuJTsgJAhkAtSepSZkq0Zn4FDVtvZyASWzovR2OeeyiAA==",
+                //temporally disabled yum cannot install ifconfig
+                "embedded.memsql.install.enabled=false"
```

Comment:
check `EmbeddedKafkaTestOperationsAutoConfiguration` how test operations should be disabled